armi/reactor/blockParameters.py

```diff
@@ -887,21 +899,21 @@ def xsTypeNum(self, value):
 
         pb.defParam(
             "deltaTclad",
-            units="1/cm^5/s^2",
+            units=f"1/{units.CM}^5/{units.SECONDS}^2",
```

Comment:
huh

This looks like it's just "delta T".  So, the units should be temperature, right?

Reply:
Looking in the internal plugin that uses this, it would appear that it is a slope. Something like `dT/dP` where `P` is power in units of percent.

@mgjarrett can maybe confirm.

Also, this parameter doesn't belong in the framework, it should definitely be in the internal plugin itself... 😞

Reply:
I see "K / %pow" in some downstream repos, for these 3 parameters.

But in that case, it wouldn't matter if power was in Watts, kW, or even Horse Power.  Since it's just a percentage change in power.  

So I believe the units are just `K / %`.